### PR TITLE
Adds `@learn /breaking-changes` AI-assistant command

### PR DESCRIPTION
**Pull request by IEvangelist**
Adds `@learn /breaking-changes` AI-assistant command

image

image

image

image

image

### CODE REVIEW

File: src/consts.ts [hunk 1]
```diff
@@ -62,3 +63,42 @@ export const allUrlFormatQuickPickItems: QuickPickItem[] =
...
+
+export function getBreakingChangePrompt(issue: Issue): string {
+    return `Please create a Markdown file that contains the breaking changes from the following GitHub issue:
+    
+    "${issue.body}"
+
+    You'll need to write a file with the following, a markdown frontmatter similar to:
+    ---
+    title: "${issue.title}"
+    description: <TODO: Summarize the article>
```
Comment (gewarren) [inline src/consts.ts:82]: Should we put the optimal number of characters (from Learn Linter)?

File: src/consts.ts [hunk 2]
```diff
@@ -62,3 +63,42 @@ export const allUrlFormatQuickPickItems: QuickPickItem[] =
...
+    ---
+
+    And then the following sections:
+
+    - h1: "${issue.title}"
+      A brief summary of the breaking change.
+    - h2: Version introduced
+      A single phrase, such as .NET Aspire 9.0 GA.
+    - h2: Previous behavior
+      A brief description of the behavior before the change.
```
Comment (gewarren) [inline src/consts.ts:93]:
```suggestion
      A brief description of the behavior before the change, including a code snippet if applicable.
```

File: src/consts.ts [hunk 3]
```diff
@@ -62,3 +63,42 @@ export const allUrlFormatQuickPickItems: QuickPickItem[] =
...
+    - h2: New behavior
+      A brief description of the behavior after the change.
```
Comment (gewarren) [inline src/consts.ts:95]:
```suggestion
      A brief description of the behavior after the change, including a code snippet if applicable.
```

File: src/consts.ts [hunk 4]
```diff
@@ -62,3 +63,42 @@ export const allUrlFormatQuickPickItems: QuickPickItem[] =
...
+    - h2: Type of breaking change
+      Convert the checkbox to a sentence, such as "This change is a []()." where the link points to the appropriate category in the categories.md file.
+    - h2: Recommended action
+      A brief description of the action that users should take
```
Comment (gewarren) [inline src/consts.ts:99]:
```suggestion
      A brief description of the action or actions that users should take, including code snippets if applicable.
```

File: src/consts.ts [hunk 5]
```diff
@@ -62,3 +63,42 @@ export const allUrlFormatQuickPickItems: QuickPickItem[] =
...
+    - h2: Affected APIs
+      A list of APIs that are affected by the change. If there are no affected APIs (or "No response") write "None.".
```
Comment (gewarren) [inline src/consts.ts:101]:
```suggestion
      A list of APIs, in xref format, that are affected by the change. If there are no affected APIs (or "No response") write "None.".
```

File: src/consts.ts [hunk 6]
```diff
@@ -62,3 +63,42 @@ export const allUrlFormatQuickPickItems: QuickPickItem[] =
...
+
+    Always reply with Markdown. Use active voice, and write in the present tense. When writing the Reason for the change section, write it in this format: "This change is a [sentence case name](../categories.md#book-mark).`;
```
Comment (gewarren) [inline src/consts.ts:103]: This sounds more like the type of breaking change, not reason?

**Review by gewarren — APPROVED** [review]

File: src/consts.ts (hunk 1 above)
Comment (IEvangelist) [reply src/consts.ts:82]: Is it 160 characters?

File: src/consts.ts (hunk 6 above)
Comment (IEvangelist) [reply src/consts.ts:103]: Ah, good catch. I had mixed these up a bit... This was intended for the Type of change heading, not the reason. https://learn.microsoft.com/en-us/dotnet/core/compatibility/containers/9.0/no-zlib#type-of-change

File: src/consts.ts (hunk 1 above)
Comment (gewarren) [reply src/consts.ts:82]: Between 100 and 160.

Comment (gewarren) [reply src/consts.ts]: And title is 19-59 characters.

**Pull request merged**